rero/rero-ils
# ils: translates v0.5.0 strings

* NEW Translates strings for fees resources and holdings for harvested
  ebooks.
* BETTER Harmonizes strings between api, schemas and form options.
* FIX Fixes wrong end of line inside strings to be translated
  (javascript), in the circulation policies editor.
* FIX Fixes typo in a string on the frontpage.

Co-Authored-by: Igor Milhit <igor.milhit@rero.ch>

## Why are you opening this PR?
- Because translations weren't completely done before next release (`v0.5.0`).

## How to test?

- Travis is green
- `./run-tests.sh`
- Open circulation policies editor, item_type editor and location editor and verify it's translated.

## Code review check list
